src/models/PublicGroupGoalItem.ts

```diff
@@ -0,0 +1,10 @@
+export interface PublicGroupGoalItem {
+    feelingScore: number | null,
```

Comment:
its better to avoid null as it adds an additional check whenever we try to use the value